src/components/FolderPage/FolderBar.js

```diff
@@ -0,0 +1,141 @@
+import styled from 'styled-components';
+import { GRAY4, PRIMARY, WHITE } from '../color';
+import Add from '../../asset/add.svg';
+import AddWhite from '../../asset/addWhite.svg';
+import FolderNameBar from './FolderNameBar';
+
+const Block = styled.button`
+  display: flex;
+  align-items: center;
+  height: 35px;
+  font-family: Pretendard;
+  font-size: 16px;
+  padding: 8px 12px;
+  border-radius: 5px;
+  border: 1px solid ${PRIMARY};
+
+  &:hover {
+    background-color: ${GRAY4};
+    transition: all 300ms ease-in-out;
+  }
+
+  @media (max-width: 767px) {
+    padding: 6px 10px;
+    font-size: 14px;
+    height: 29px;
+  }
+
+  ${({ $clicked }) =>
+    $clicked &&
+    `background-color: ${PRIMARY};
+  color: ${WHITE};
+  &:hover{
+    background-color: ${PRIMARY};
+  }`}
+`;
+
+const FolderList = styled.ul`
+  display: flex;
+  gap: 12px 8px;
+  flex-wrap: wrap;
+`;
+
+const Bar = styled.div`
+  display: flex;
+  column-gap: 12px;
+  justify-content: space-between;
+  align-items: center;
+`;
+
+const AddFolder = styled.button`
+  color: ${PRIMARY};
+  display: flex;
+  align-items: center;
+  font-family: Pretendard;
+  flex-shrink: 0;
+  font-size: 16px;
+  font-weight: 500;
+  letter-spacing: -0.3px;
+
+  &::after {
+    content: '';
+    background-image: url(${Add});
+    background-size: cover;
+    display: inline-block;
+    width: 16px;
+    height: 16px;
+    margin-left: 4px;
+  }
+
+  @media (max-width: 767px) {
+    color: ${WHITE};
+    padding: 8px 24px;
+    position: fixed;
+    left: 50%;
+    transform: translateX(-50%);
+    bottom: 101px;
+    border-radius: 20px;
+    border: 1px solid ${WHITE};
+    background: ${PRIMARY};
+    z-index: 4;
+
+    &::after {
+      background-image: url(${AddWhite});
+    }
+  }
+`;
+
+const FolderBlock = ({ item, clicked, children }) => {
+  return (
+    <Block name={item.id} $clicked={clicked} type="button">
+      {children}
+    </Block>
+  );
+};
+
+const FolderIndex = ({ items, onClick, selectedId }) => {
+  return (
+    <FolderList onClick={onClick}>
+      <li key={-1}>
+        <FolderBlock item={{ id: '-1' }} clicked={selectedId ? false : true}>
+          전체
+        </FolderBlock>
+      </li>
+      {items?.map(function (item) {
+        let clicked;
+        if (item?.id === selectedId) {
+          clicked = true;
+        } else {
+          clicked = false;
+        }
+        return (
+          <li key={item?.id}>
+            <FolderBlock item={item} clicked={clicked}>
```

Comment:
아래처럼 간결히 작성할 수 있을 것 같습니다~! 

```suggestion
            <FolderBlock item={item} clicked={item?.id === selectedId}>
```